### DISCUSSION TITLE
Routing package

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#197] [General]
Can you suggest a flutter routing package that fits best with ReArch philosophy?

I'm have been using Beamer lately, and even If i could like it I find it very limited. (I'm just forked it and checking if I can do something on it).

**Comment by GregoryConrad** [comment]
Not sure what would best fit. I’ve used auto_route before and like it, but a lot of folks use go_router

**Reply by busslina** [reply]
Never used `auto_route` yet. Maybe gonna study it
